config/checker/com.intellij.astub

```diff
@@ -1,12 +1,18 @@
-
 import org.checkerframework.checker.guieffect.qual.AlwaysSafe;
 import org.checkerframework.checker.guieffect.qual.SafeEffect;
 import org.checkerframework.checker.guieffect.qual.UI;
 import org.checkerframework.checker.guieffect.qual.UIEffect;
 import org.checkerframework.checker.guieffect.qual.UIPackage;
 import org.checkerframework.checker.index.qual.NonNegative;
+import com.intellij.codeInspection.util.InspectionMessage;
 import org.checkerframework.checker.initialization.qual.UnknownInitialization;
 import org.checkerframework.checker.nullness.qual.Nullable;
+import org.jetbrains.annotations.NotNull;
```

Comment:
Imports from outside `org.checkerframework.checker.*` packages are almost surely not needed in astubs... lemme see the specific usage, but in general almost everything is ignored by Checker in astubs, other than Checker's own annotations

Reply:
hmm but if I e.g. delete @Nullable annotation from GitUntrackedFilesHelper.notifyUntrackedFilesOverwrittenBy and description param I am getting an exception
<img width="1686" alt="Screenshot 2022-06-09 at 17 36 25" src="https://user-images.githubusercontent.com/33667003/172887300-32912708-dc04-4a5f-9fd0-c20a7a2bb24d.png">

Reply:
But when I add @Nullable to stub (as it is in original declaration) it works properly

Reply:
Poor \@nullable, they must be mentioned 1000s times a day 😅 tip: you can use `\@nullable`

Reply:
Yeah, so `@Nullable` annotations are to be left, `@NotNull` to be removed as redundant